### PR TITLE
added instructions and files

### PR DESCRIPTION
**Pull request by andysotocruz**

### CODE REVIEW

File: README.md
Comment (eamzea) [inline README.md:1]: What have I told you about mixing things?
If you're writing in English write everything in English. If your are writing in Spanish write everything in Spanish.

Maybe you can write all the instructions in Spanish and add the translation after.

File: README.md [hunk 1]
```diff
@@ -1 +1,62 @@
-# movies-description
\ No newline at end of file
+## movies-description instrucciones:
+
+_Hola David en este repositorio trabajaras en la carpeta de **all-movies**, dentro de la carpeta **movies** encontraras otras 3 carpetas que llaman **first-movie, second-movie, third-movie** dentro de estas carpetas hay un archivo llamado **index.html** en este archivo pondrás la descripción de la película, en la carpeta **assets** meterás la imagen de la película que escogiste. A continuación te dejaré los pasos a seguir para que trabajes en el repositorio._
```
Comment (eamzea) [inline README.md:3]: trabajarás*
encontrarás*

File: README.md [hunk 2]
```diff
@@ -1 +1,62 @@
...
+1. a continuación dejaré un link de una página en la cual podrás buscar tus películas:
+
+[![allmovies](https://www.themoviedb.org/assets/2/v4/logos/v2/blue_short-8e7b30f73a4020692ccca9c88bafe5dcb6f8a62a4c6bc55cd9ba82bb2cd95f6c.svg)](https://www.themoviedb.org/?language=es-es)
+
+2. al entrar a la página, buscarás el nombre de la película que gustes ejemplo: **_13 horas: Los soldados secretos de Bengasi**.
+
+3. te arrojará una lista de películas y darás clic en la película.
+
+>Ejemplo: ![Alt text](<Captura de Pantalla 2023-07-03 a la(s) 17.38.23.png>)
+
```
Comment (eamzea) [inline README.md:27]: You're giving more space

File: README.md
Comment (eamzea) [inline README.md:1]: Every step should start with a capitalized letter

File: README.md [hunk 3]
```diff
@@ -1 +1,62 @@
...
+1. al terminar de subir cambios deberás hacer un `PR (pull request)`desde github.
+
+2. en el titulo del `PR`deberás poner un titulo.
+
+>ejemplo: ***union de primer pelicula***
+
+3. hacer el merge a la rama `develop`.
+
+4. al terminar de unir todas las ramas, deberás actualizar la rama `develop` de **GitHub**
```
Comment (eamzea) [inline README.md:60]: Update GitHub develop branch?

File: README.md [hunk 4]
```diff
@@ -1 +1,62 @@
...
+
```
Comment (eamzea) [inline README.md:61]: Maybe you can add one step:
- Add me or Edgar as reviewer
- Once the PR is approved merge the PR

**Review by eamzea — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -1 +1,67 @@
-# movies-description
\ No newline at end of file
+## películas-descripción instrucciones:
+
+_Hola David en este repositorio trabajarás en la carpeta de **all-movies**, dentro de la carpeta **peliculas** encontrarás otras 3 carpetas que llaman **primera-pelicula, segunda-pelicula, tercera-pelicula** dentro de estas carpetas hay un archivo llamado **index.html** en este archivo pondrás la descripción de la película, en la carpeta **assets** meterás la imagen de la película que escogiste. A continuación te dejaré los pasos a seguir para que trabajes en el repositorio._
```
Comment (eamzea) [inline README.md:3]: **otras 3 carpetas que llaman** ?

File: README.md [hunk 6]
```diff
@@ -1 +1,67 @@
...
+3. Bajar actualizaciones de la rama `develop`.
+
+---
+## Segundo bloque.
+
+1. A continuación dejaré un link de una página en la cual podrás buscar tus películas:
+
+[![allmovies](https://www.themoviedb.org/assets/2/v4/logos/v2/blue_short-8e7b30f73a4020692ccca9c88bafe5dcb6f8a62a4c6bc55cd9ba82bb2cd95f6c.svg)](https://www.themoviedb.org/?language=es-es)
+
+2. Al entrar a la página, buscarás el nombre de la película que gustes ejemplo: **_13 horas: Los soldados secretos de Bengasi**.
```
Comment (eamzea) [inline README.md:22]: película que gustes  _**,**_ ejemplo

Missing coma

File: README.md [hunk 7]
```diff
@@ -1 +1,67 @@
...
+>Ejemplo: ![Alt text](<Captura de Pantalla 2023-07-03 a la(s) 17.38.23.png>)
+
+4. Al entrar a la vista de la película, descargarás la imagen que pone como cartelera.
+
+---
+## Tercer bloque.
+
+1. Añadir la imagen de la película descargada en la carpeta de assets.
+
+2. En la carpeta **primera-pelicula** en el archivo de ***index.html***, pondrás la información de la película en forma de listado sin espacios, y al menos 2 o 3 nombres de los protagonistas de la película y copiar y pegar la sinopsis ejemplo:
```
Comment (eamzea) [inline README.md:35]: En la carpeta **primera-pelicula** en el archivo de ***index.html***, pondrás la información de la película en forma de listado, sin espacios y al menos 2 o 3 nombres de los protagonistas de la película, copiar y pegar la sinopsis ejemplo:

File: README.md [hunk 8]
```diff
@@ -1 +1,67 @@
...
+
+1. Al terminar de subir cambios deberás hacer un `PR (pull request)`desde github.
+
+2. En el titulo del `PR`deberás poner un titulo.
+
+>ejemplo: ***union de primer pelicula***
+
+3. Hacer la comparación de tu rama con `develop`.
+
+4. deberás agregar a Edgar o Andres en la parte de **reviewers**, para revisar el `PR` ,darás click en el engrane que esta en la parte derecha de **reviewers** y seleccionar a cual quiera de las 2 personas antes mencionadas.
```
Comment (eamzea) [inline README.md:59]: Capitalized

Comment (eamzea) [inline README.md:59]: cualquiera*

File: README.md [hunk 9]
```diff
@@ -1 +1,67 @@
...
+
+>ejemplo ![Alt text](<Captura de Pantalla 2023-07-03 a la(s) 19.27.30.png>)
+
+5. Una vez aprobado el `PR (pull request)`deberás hacer click en **merge pull reques**
```
Comment (eamzea) [inline README.md:63]: request*

File: README.md [hunk 10]
```diff
@@ -1 +1,67 @@
...
+
+6. Al terminar de unir todas las ramas, deberás actualizar la rama `develop`.
```
Comment (eamzea) [inline README.md:65]: If you don't want to write "locally" you can say:
Once al branches are merged you should update **your** develop branch

**Review by eamzea — CHANGES_REQUESTED** [review]

File: README.md [hunk 11]
```diff
@@ -1 +1,67 @@
...
+3. Al terminar de agregar la información, deberás guardar, comentar y subir cambios
+
+---
+## Cuarto bloque
+
+1. Al terminar de subir cambios deberás hacer un `PR (pull request)` desde github.
+
+2. En el titulo del `PR` deberás poner un titulo.
+
+>ejemplo: ***union de primer pelicula***
```
Comment (eamzea) [inline README.md:55]: If all examples are capitalized this is one is missing

File: README.md (hunk 4 above)
Comment (eamzea) [reply README.md:66]: This is not solved

File: README.md (hunk 11 above)
Comment (eamzea) [inline README.md:55]: unión*

File: Captura de Pantalla 2023-07-03 a la(s) 17.38.23.png
Comment (eamzea) [inline Captura de Pantalla 2023-07-03 a la(s) 17.38.23.png:1]: You should rename this SS to be more descriptive

File: Captura de Pantalla 2023-07-03 a la(s) 19.27.30.png
Comment (eamzea) [inline Captura de Pantalla 2023-07-03 a la(s) 19.27.30.png:1]: You should rename this SS to be more descriptive

**Review by eamzea — CHANGES_REQUESTED** [review]

File: README.md (hunk 10 above)
Comment (eamzea) [reply README.md]: This one is not solved

Comment (andysotocruz) [reply README.md]: my friend this problem has been solved

File: README.md (hunk 4 above)
Comment (andysotocruz) [reply README.md:66]: my friend this problem has been solved

File: Captura de Pantalla 2023-07-03 a la(s) 17.38.23.png
Comment (eamzea) [reply Captura de Pantalla 2023-07-03 a la(s) 17.38.23.png:1]: This is not solved

File: Captura de Pantalla 2023-07-03 a la(s) 19.27.30.png
Comment (eamzea) [reply Captura de Pantalla 2023-07-03 a la(s) 19.27.30.png:1]: This is not solved

File: README.md [hunk 12]
```diff
@@ -1 +1,67 @@
-# movies-description
\ No newline at end of file
+## películas-descripción instrucciones:
```
Comment (eamzea) [inline README.md:1]: File title should be capitalized

File: README.md [hunk 13]
```diff
@@ -1 +1,67 @@
...
+3. Bajar actualizaciones de la rama `develop`.
+
+---
+## Segundo bloque.
+
+1. A continuación dejaré un link de una página en la cual podrás buscar tus películas:
+
+[![allmovies](https://www.themoviedb.org/assets/2/v4/logos/v2/blue_short-8e7b30f73a4020692ccca9c88bafe5dcb6f8a62a4c6bc55cd9ba82bb2cd95f6c.svg)](https://www.themoviedb.org/?language=es-es)
+
+2. Al entrar a la página, buscarás el nombre de la película que gustes, ejemplo: **13 horas: Los soldados secretos de Bengasi**.
```
Comment (eamzea) [inline README.md:22]: Why this example is different to others?

File: README.md [hunk 14]
```diff
@@ -1 +1,67 @@
...
+2. En la carpeta **primera-pelicula** en el archivo de ***index.html***, pondrás la información de la película en forma de listado, sin espacios y al menos 2 o 3 nombres de los protagonistas de la película, copiar y pegar la sinopsis ejemplo:
+
+- Nombre: 13 horas los soldados de Bengasi
+- Director: Michael Bay
+- Año de estreno: 2016
+- Género: Guerra, Acción, Historia, Drama.
+- Puntuación de usuario: 73%
+- Protagonista 1: Juan Krasinski
+- Protagonista 2: James Insignia Dale
+- Sinopsis: Cuando todo iba mal, seis hombres tuvieron el valor de hacer lo correcto. El 11 de septiembre de 2012, en el aniversario de los ataques terroristas a las Torres Gemelas y al Pentágono, un grupo de milicianos islamistas atentaron contra el consulado estadounidense y un anexo cercano de la CIA en Bengasi, Libia. Un equipo de seis miembros de Operaciones Especiales de Estados Unidos fue enviado para rescatar a los supervivientes.
```
Comment (eamzea) [inline README.md:44]: This example is also different

**Review by eamzea — CHANGES_REQUESTED** [review]
You have 5 SS
Do you need all of them?

File: README.md (hunk 13 above)
Comment (andysotocruz) [reply README.md:22]: this is not an example, it is an image with the link to the movie page

Comment (andysotocruz) [reply README.md:22]: referring to the SS, there are not 5 SS in reality there are only 2 and one is the logo of the page of the movies.

Comment (eamzea) [reply README.md:22]: Al entrar a la página, buscarás el nombre de la película que gustes, **ejemplo**: **13 horas: Los soldados secretos de Bengasi**.

I'm not reading very well or here it says "ejemplo" ?

**Review by eamzea — CHANGES_REQUESTED** [review]

**Review by eamzea — CHANGES_REQUESTED** [review]
> referring to the SS, there are not 5 SS in reality there are only 2 and one is the logo of the page of the movies.

Are you sure there are only 2 SS ?

Screenshot 2023-07-03 at 21 07 31

Screenshot 2023-07-03 at 21 11 02

Then why in the branch appear 5 .png files? And why the PR shows 5 .png files?

I'm getting blind?

**Review by eamzea — APPROVED** [review]
LGTM!

**Pull request merged**